app/src/main/java/com/example/mywatchlist/BaseApplicationForHilt.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.mywatchlist
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+
+@HiltAndroidApp
+class BaseApplicationForHilt: Application() {
```

Comment:
Change name of the application to something better

Reply:
What about Watchlist?